backend/src/main/java/gov/cdc/usds/simplereport/config/DataHubClientConfiguration.java

```diff
@@ -9,14 +9,17 @@ public class DataHubClientConfiguration {
   @Value("${datahub.api-key}")
   private String apiKey;
 
-  private static final String CSV_API_VERSION = "2021-09-21";
-  private static final String SIMPLE_REPORT_CLIENT_NAME = "simple_report";
+  @Value("${datahub.api-version}")
+  private String csvApiVersion;
+
+  @Value("${datahub.api-client}")
+  private String simpleReportClientName;
```

Comment:
Can we rename this to simpleReportCsvUploadClientName and same for the yaml value? since this will be different from the regular client name for SR